appveyor.yml

```diff
@@ -13,7 +13,8 @@ environment:
       PYTHON_VERSION: 3.6
     - PYTHON: "C:\\Miniconda37-x64\\Scripts\\activate.bat"
       PYTHON_VERSION: 3.7
-
+    - PYTHON: "C:\\Miniconda38-x64\\Scripts\\activate.bat"
```

Comment:
Strange that appveyor can't find this it's listed on their docs https://www.appveyor.com/docs/windows-images-software/#miniconda